client/src/components/admin/teams/TeamsView.jsx

```diff
@@ -75,15 +81,28 @@ class TeamsView extends Component {
         e.isChecked = false;
       }
     });
-    this.setState({ cohorts: newCohortList });
+    const selectedCohort = newCohortList.filter(c => c.isChecked);
+    this.setState({
+      cohorts: newCohortList,
+      selectedCohort:
+        selectedCohort.length && selectedCohort[0].name ? selectedCohort[0].name.toLowerCase() : ''
+    });
   }
 
   showDetails() {
-    console.log('TODO: display list of students for the selected cohort');
+    const { selectedCohort, studentsListByCohort } = this.state;
+    const selectedCohortDetails = studentsListByCohort.filter(
+      cohort => cohort.name === selectedCohort
+    );
+    if (selectedCohortDetails && selectedCohortDetails.length) {
+      this.setState({
+        studentsListForSelectedCohort: selectedCohortDetails[0].students
+      });
+    }
   }
 
   render() {
-    const { cohorts, activeItem } = this.state;
+    const { cohorts, activeItem, studentsListForSelectedCohort } = this.state;
```

Comment:
Try `selectedCohortStudents`  In general, we don't need to call a thing a collection (or list) if its name implies clearly that it's a collection.